projects/com.oracle.truffle.llvm.runtime/src/com/oracle/truffle/llvm/runtime/LLVMOptions.java

```diff
@@ -198,8 +199,12 @@ public static boolean isNativeCallStats() {
         return getParsedProperty(Property.NATIVE_CALL_STATS);
     }
 
+    public static boolean isNativeAnalysis() {
```

Comment:
`isNativeAnalysis` doesn't really make sense as English. Is it what? `isNativeAnalysisEnabled` maybe?